Add scraped page archive

ondenman opened the pull request:
## What does this do?

Uses scraped-page-archive to archive all pages scraped.

## Why is this needed?

There's an election coming up (01/12/2016), and it's likely that the data on the official site will disappear, meaning any data we're not already picking up will be lost. Archiving it now gives us the chance to go back and re-scrape later even if it disappears.

## Relevant Issue(s):

https://github.com/everypolitician/everypolitician-data/issues/20544

## Checklists:

### Scraper change:
* [ ] scraper is on Morph.io under the "everypolitician-scrapers" group — no, it's still at https://morph.io/tmtmtmtm/gabon-deputes, but there's little point in moving it this close to the election.
* [x] scraper's GitHub "Website" link points at morph.io page 
* [ ] scraper is set to auto-run — not yet: @tmtmtmtm This needs to be set on Morph.
* [x] scraper is configured for archiving — that's what this PR is doing!

### Adding Archiving:
* [x] scraper uses scraped-page-archive gem directly or via a suitable strategy — uses it directly
* [ ] MORPH_SCRAPER_CACHE_GITHUB_REPO_URL is configured — not yet. @tmtmtmtm this needs configured on morph
* [x] pages are being archived in new branch of correct scraper repo (yay!)

### Gemfile change:
* [x] all links are secure
* [x] links to Github use `github:` protocol, not simply `git:`
* [x] formatting is consistent with our normal Rubocop setup (See commit message.)

ondenman committed:
Add scraped page archive gem

ondenman committed:
Bundle install

ondenman committed:
Add scraped page archive

tmtmtmtm reviewed (changes requested):
Doesn't include https://github.com/everypolitician/everypolitician/wiki/Scraper-Change-checklist

ondenman committed:
Add rubocop config

ondenman committed:
Add rubocop gem

ondenman committed:
Bundle install

tmtmtmtm commented:
@ondenman the rubocop tidying only needs to be for the `Gemfile`, not the `scraper.rb`. As the autofix actually changes the logic in the scraper (by hoisting the guard clause), I'm a little hesitant to include this here. I'd suggest dropping `737f9ac`, and rerunning separately against only the Gemfile.

(NB you don't necessarily need to actually _include_ Rubocop+config in these sort of changes, but there's no harm in doing so, so now that it's already here it's OK to keep it)

ondenman committed:
Tidy Gemfile for Rubocop

ondenman committed:
Add secure git gem source

ondenman committed:
Fix Github gem source

Source was pointing to :git not :github source as defined above

ondenman committed:
Update Gemfile.lock with bundle install

tmtmtmtm committed:
Archive some other (unprocessed) pages

The only two relevant pages I could find from the outgoing legislature
were the Commissions and the list of female deputies.

tmtmtmtm commented:
The PR description here is a bit misleading — this isn't really archiving any pages, only a PDF file which presumably is never going to change. As such this change probably isn't that useful (other than a single run to archive that PDF once). 

However, I suspect we should _also_ archive (though not process) http://www.assemblee-nationale.ga/34-deputes/168-bureaux-des-commissions/ and http://www.assemblee-nationale.ga/34-deputes/153-les-femmes-deputes/ 

I've added an extra commit to pick up those two pages.

tmtmtmtm commented:
I've configured morph, though I haven't set it to run every day, I doubt it's actually going to change again between now and the election.
